### PR TITLE
option to mount/umount without calling

### PR DESCRIPTION
**Pull request by vartec**

### CODE REVIEW

File: bay/plugins/mounts.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def mounts(app):
 @click.argument('mount', type=MountType())
 @click.argument('container', default='all', type=ContainerType(all=True))
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option('--up/--no-up', '-u', default=True)
 @click.pass_obj
 def mount(app, mount, container, host):
```
Comment (ashfall-eb) [inline bay/plugins/mounts.py]: You'll need to add `up` to the args here.

File: bay/plugins/mounts.py [hunk 2]
```diff
@@ -62,14 +63,17 @@ def mount(app, mount, container, host):
...
+        app.invoke("up")
 
 
 @click.command()
 @click.argument('mount', type=MountType())
 @click.argument('container', default='all', type=ContainerType(all=True))
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option('--up/--no-up', '-u', default=True)
 @click.pass_obj
 def unmount(app, mount, container, host):
```
Comment (ashfall-eb) [inline bay/plugins/mounts.py]: You'll need to add `up` to the args here.

Comment (vartec) [inline bay/plugins/mounts.py]: done

File: bay/plugins/mounts.py [hunk 3]
```diff
@@ -51,8 +51,9 @@ def mounts(app):
 @click.argument('mount', type=MountType())
 @click.argument('container', default='all', type=ContainerType(all=True))
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option('--up/--no-up', '-u', default=True)
 @click.pass_obj
-def mount(app, mount, container, host):
+def mount(app, mount, container, host, user_profile):
```
Comment (ashfall-eb) [inline bay/plugins/mounts.py]: This should be `up`, right?

Comment (vartec) [inline bay/plugins/mounts.py]: it totally should, Sublime "autocorrected" `up` to `user_profile` :-/

**Comment by ashfall-eb** [comment]
All good, merging thanks!

**Pull request merged**